client/src/components/matrix/Matrix.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { useEffect, useState } from "react";
+import { colors } from "../../pages/colors.js";
+import { SelectableGroup } from "react-selectable";
```

Comment:
Use  @simonwep/selection-js library: https://simonwep.github.io/selection/ as I told you in our last meeting. Make appropriate adjustments in the code.